documentation/api/change_log.rst

```diff
@@ -5,6 +5,10 @@ API change log
 
 .. note:: The FlexMeasures API follows its own versioning scheme. This is also reflected in the URL, allowing developers to upgrade at their own pace.
 
+v3.0-8 | 2023-08-01
+"""""""""""""""""""
+
+- Added REST endpoint for patching a sensor: `/sensor<id>` (PATCH)
```

Comment:
I would also summarize these three here, for easier reading.

Reply:
updated as one change.